### ISSUE TITLE
[BUG] deployment fail on first attempt while following training wheels

### ISSUE THREAD

**Issue by NorseL0rd** [nmelhado/league-page#294]
**Describe the bug**
<html>
<body>

error during build:
--
20:17:05.288 | Error: Unsupported Node.js version: v20.11.1. Please use Node 16 or Node 18 to build your project, or explicitly specify a runtime in your adapter configuration.

</body>
</html>

**Your League ID** 1050069781250048000
In many cases, the bug maybe related to something not yet accommodated for in League Page, or something specific to your league. Adding your league ID allows for a faster debug and bug fix.

**To Reproduce**
1. First try at deploy in training wheels and received error 

**Expected behavior**

Successful deploy

**Screenshots**
![image](https://github.com/nmelhado/league-page/assets/163696211/7c14fcca-f8eb-4d4c-8ef6-dbe5feade02f)

**Desktop (please complete the following information):**
 - OS: [Windows]
 - Browser [Edge]
 - Version [Version 122.0.2365.92 (Official build) (64-bit)

**Additional context**

**Comment by scanale42** [comment]
Have the same problem. Deploying works fine - I'm able to see the league page after I change the league ID. when I go and edit the the leagueinfo.js again and commit changes to the main page - IE - just the text on the home page, After I hit commit, I start getting these same errors on the build.

**Comment by scanale42** [comment]
Played around with this a bit. Tried changing the version in vercel to using 20.x, and matching that in the packages.json. That didn't do anything. Eventually changed it back to 18.x in vercel and edting the like in packages.json to say:  "node": "18.x", Once I did this - that fixed the problem and I was able to make changes to the leagueinfo.js file again and have the build complete successfully every time on vercel and reflect on the league home page.
